src/LCT.Common/Constants/Dataconstant.cs

```diff
@@ -14,6 +15,15 @@ namespace LCT.Common.Constants
     [ExcludeFromCodeCoverage]
     public static class Dataconstant
     {
+        private static Dictionary<string, string> purlids = new Dictionary<string, string>
+         {
```

Comment:
Since you added all the pkg variables in a dictionary, is the references for this changed for every package type?

Reply:
public static Dictionary<string, string> PurlCheck()
        {
            return purlids;
        }

for this method it's there